### PR TITLE
[SPEED MERGE NEEDED] removes cat

### PR DESCRIPTION
**Pull request by pireamaineach**
I got maintainer permission however I did ask two coders simultaneously and they gave me different reactions, which feels rather scummy in retrospect. A.N. advised against it but stylemistake said to go ahead.

## About The Pull Request

cats make me pp hard

## Why It's Good For The Game

I don't want another vote next year.

## Changelog
:cl:
del: cat gone
/:cl:

### CODE REVIEW

**Comment by TheVekter** [comment]
1) No
2) You tried to get 48000 but you *failed*

**Comment by TheVekter** [comment]
wait I almost forgot

https://media.discordapp.net/attachments/254009936080666624/626917410485829681/Roach.gif

**Comment by pireamaineach** [comment]
I wasn't paying attention to the PR number.

**Comment by TheChosenEvilOne** [comment]
All species related PRs require a maintainer approval, right?

**Comment by pireamaineach** [comment]
You did not read

**Comment by stylemistake** [comment]
> cats make me pp hard

Not a problem of cats. Remove PP.

**Comment by pireamaineach** [comment]
> > cats make me pp hard
> 
> Not a problem of cats. Remove PP.

I don't have pp

**Comment by TheChosenEvilOne** [comment]
> 
> 
> You did not read

I can't read.

**Comment by TheChosenEvilOne** [comment]
> 
> 
> > > cats make me pp hard
> > 
> > 
> > Not a problem of cats. Remove PP.
> 
> I don't have pp

So, what is the issue then?

**Comment by pireamaineach** [comment]
They make pp hard

**Comment by TheChosenEvilOne** [comment]
> 
> 
> They make pp hard

who

**Comment by Takov-Mala** [comment]
DO IT
Enough of this Brexit shit with revotes until certain people get the result they like. 
WE VOTED LEAVE (the catshits to die in agony outside the airlocks)

**Comment by KomradeSpectre** [comment]
Oh damn, Takov showed up, we're nearing completion of the cursed trifecta. We need a @dee-dubya comment stat!

**Comment by TheVekter** [comment]
in before the lock

**Comment by pireamaineach** [comment]
What am I supposed to post. @stylemistake

**Comment by KomradeSpectre** [comment]
@stylemistake 
image

**Comment by pireamaineach** [comment]
Pictures of nothing because cats are gone.

**Comment by TheVekter** [comment]
image

**Comment by Naloacisamazing** [comment]
This PR got them 
thisfuckinPR

Vibes

**Comment by pireamaineach** [comment]
This fellow seems to not like the felinids of which make my pp hard. He is a clown, yes, a clown. I shall post a clown picture to show him as such.

File: code/game/objects/items/devices/laserpointer.dm [hunk 1]
```diff
@@ -134,7 +134,7 @@
 
 	//catpeople
 	for(var/mob/living/carbon/human/H in view(1,targloc))
-		if(!isfelinid(H) || H.incapacitated() || H.eye_blind )
+		if( H.incapacitated() || H.eye_blind )
```
Comment (Dorsisdwarf) [inline code/game/objects/items/devices/laserpointer.dm]: This makes all humans grab for the laser pointers

Comment (pireamaineach) [reply code/game/objects/items/devices/laserpointer.dm]: It makes them what for the laser pointers

Comment (KomradeSpectre) [reply code/game/objects/items/devices/laserpointer.dm]: Good change tbh

**Comment by actioninja** [comment]
holy crap lois!

**Comment by KomradeSpectre** [comment]
Waiting for the boomerang

**Comment by KomradeSpectre** [comment]
There it is

**Comment by redmoogle** [comment]
Blessed

**Comment by pireamaineach** [comment]
It's alright new PR time

**Comment by TheVekter** [comment]
Nice meme

**Comment by FancyCyclops** [comment]
Divide and Conquer.

**Comment by MCterra10** [comment]
@actionija you didn't revert the changelog lmao

**Comment by optimumtact** [comment]
it's only funny when I do this @actioninja

**Comment by pireamaineach** [comment]
You should revert the revert to punish them. @optimumtact

**Comment by optimumtact** [comment]
you are about a micromillimeter away from being banned permanently, you are not in a position to suggest anything

**Comment by pireamaineach** [comment]
Rude.

File: code/game/objects/items/devices/laserpointer.dm (hunk 1 above)
Comment (MadmanMartian) [reply code/game/objects/items/devices/laserpointer.dm]: How do you keep doing this? I fixed this before

**Pull request merged**